include/picongpu/plugins/radiation/Radiation.hpp

```diff
@@ -1188,8 +1209,8 @@ class Radiation : public ISimulationPlugin
       PMACC_KERNEL( KernelRadiationParticles<
           numWorkers
       >{} )(
-          gridDim_rad,
-          numWorkers
+          DataSpace< 2 >(gridDim_rad, numJobs),
+          DataSpace< 2 >(numWorkers,1)
```

Comment:
Here again `2`

Reply:
see #3354 (comment)
It is not a spatial dimension, to not mix it with the spatial  2D and 3D which is supported by picongpu I explicit set it to `2` instead of `DIM2`